rad/prelude/test-eval.rad

```diff
@@ -1,33 +1,28 @@
 {:module 'prelude/test-eval
  :doc
-   "Provides eval that adds a `:test` macro. If an the eval encounters an
+ "Provides eval that adds a `:test` macro. If an the eval encounters an
    expression of the form `(:test \"desc\" tests...)` we add it to the vector
    stored in the `tests` ref. The tests are then run by `rad/prelude/test.rad`."
- :exports '[eval tests]}
+ :exports '[tx tests]}
 
 (def tests
   "Reference that collects tests defined with the `:test` macro. This
   value is updated by a custom `eval`."
   (ref []))
 
 ;; Annoyingly verbose since we have neither the prelude nor pattern-matching.
-(def eval
-  "Evaluation function that adds :test macro to register tests."
+(def tx
+  "Transaction function that adds :test macro to register tests."
```

Comment:
In primFns it is called `transactor function` Maybe change one of them for consistency.

Reply:
Actually this changes to a macro in the macros PR so I will just leave this.